packages/libs/eda/src/lib/core/utils/default-axis-range.ts

```diff
@@ -24,7 +24,7 @@ export function numberDateDefaultAxisRange(
             min:
               logScale &&
               observedMin != null &&
-              (observedMin <= 0 ||
+              ((observedMin as number) <= 0 ||
```

Comment:
This was just a tidy-up. My IDE is on a higher version of TypeScript, and it didn't like it.